apps/test/index.js

```diff
@@ -1,3 +1,4 @@
 require('./code-studio-tests');
 require('./unit-tests');
 require('./integration-tests');
+require('./tutorialExplorer-tests');
```

Comment:
I would think these tests should just be part of unit tests, instead of creating a new grouping of tests.